jupyter-extension/jupyterlab_pachyderm/tests/test_config.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for the config screen and handler."""
+from pathlib import Path
+
+import pytest
+from httpx import AsyncClient
+from tornado.web import Application
+
+from pachyderm_sdk.config import ConfigFile
+
+from jupyterlab_pachyderm.env import PACH_CONFIG
+
+
+@pytest.mark.no_config
+async def test_config_no_file(app: Application, http_client: AsyncClient):
+    """Test that if there is no config file present, the extension does not
```

Comment:
TIOLI: to me this test seems to more test that the server correctly returns an `INVALID` cluster status rather than testing this negative statement

Reply:
Yeah I think there's some truth to what you're saying. Really its testing that no default config is set (pachd_address == "") and that the cluster status is INVALID. I'll update the docstring to better document this